### DISCUSSION TITLE
Can't search bookmarks.

### DISCUSSION THREAD

**Discussion by thetain** [sumatrapdfreader/sumatrapdf#4115] [General]
Is there a way to search by bookmarks, please?

**Comment by GitHubRulesOK** [comment]
Basically no but the bookmarks Outline should be abbreviated page contents or duplicated by a Real "Table Of Contents" or "Index" link so the more unique the letters you type the faster to get there.

**Comment by kjk** [comment]
You can now search bookmarks in latest [pre-release](https://www.sumatrapdfreader.org/prerelease)
